app/packs/javascripts/dynamic_elements/topic.js

```diff
@@ -171,6 +171,44 @@ export default class Topic extends ShikiEditable {
         this.$commentsHider.show();
       }
     });
+
+    this._bindReturnClick();
+  }
+
+  _showReturnButton() {
+    if ($('.b-comments').height() > $(window).height()) {
+      $('#return-to-reply').show();
+    }
+  }
+
+  _saveRepliablePosition(commentId) {
+    const repliablePositionKey = 'repliableId_' + window.location.pathname;
+    window.sessionStorage.setItem(repliablePositionKey, commentId);
+  }
```

Comment:
нет необходимости сохранять в sessionStorage комментарий, и вообще reply могли нажать у топика
давай в текущем объекте просто сохранять dom node, по которому кликнули

```suggestion
  _saveRepliablePosition(repliedNode) {
    this.repliedNode = repliedNode;
  }
```